test/test_VTK.jl

```diff
@@ -46,49 +46,33 @@ using SHA
                                        ("tetras", coords_3D, tetra_mesh_edof, VTKCellTypes.VTK_TETRA))
 
         vtkfile = vtk_grid(name, coord, edof, ctype)
```

Comment:
This line segfaults... But only if this file is ran with `include`, copy pasting the full testset in the REPL passes just fine.